Something/WebApplication/wwwroot/js/apps/authentication/components/Splash/Splash.scss

```diff
@@ -0,0 +1,19 @@
+.Splash {
+    background: #ddd;
+    color: #fff;
+    background-image: url(/images/splash1.jpeg);
+    background-position: center;
+    height: 100%;
+    &--login {
+        background-position-x: 25%;
+        background-position-y: 60%;
+    }
+    &--register {
+        background-position-x: 90%;
+        background-position-y: 60%;
+    }
+    &--inner {
+        width: 400px;
+        margin: 0 auto;
+    }
+}
```

Comment:
Replace `--` with `__`

Reply:
`--` is for modifiers
`__` is for block

Reply:
Thanks.